### ISSUE TITLE
Jackett in a crash loop

### ISSUE THREAD

**Issue by hhazeley** [Jackett/Jackett#5321]
Jacket will not start in a start up loop

![image](https://user-images.githubusercontent.com/22698784/58306170-84118400-7daf-11e9-8f17-b92a4eeb108e.png)

Abstract from Logs:

```
2019-05-23 22:57:15.1016 Info Starting Jackett v0.11.361.0
2019-05-23 22:57:18.5965 Info Environment version: 4.0.30319.42000 (/opt/Jackett/)
2019-05-23 22:57:18.5965 Info OS version: Unix 4.4.0.148 (64bit OS) (64bit process)
2019-05-23 22:57:18.6004 Info Jackett variant: CoreLinuxAmdx64
2019-05-23 22:57:18.6004 Info ThreadPool MaxThreads: 32767 workerThreads, 1000 completionPortThreads
2019-05-23 22:57:18.6004 Info App config/log directory: /home/msacc/.config/Jackett
2019-05-23 22:57:18.6004 Info issue: Ubuntu 16.04.6 LTS \n \l
2019-05-23 22:57:18.6127 Info Using HTTP Client: HttpWebClient2NetCore
2019-05-23 22:57:20.5487 Error System.BadImageFormatException: Could not load file or assembly 'YamlDotNet, Version=6.0.0.0, Culture=neutral, PublicKeyToken=ec19458f3c15af5e'. The module was expected to contain an assembly manifest.
File name: 'YamlDotNet, Version=6.0.0.0, Culture=neutral, PublicKeyToken=ec19458f3c15af5e'
   at Jackett.Common.Services.IndexerManagerService.InitCardigannIndexers(IEnumerable`1 path)
   at Jackett.Common.Services.IndexerManagerService.InitIndexers(IEnumerable`1 path) in /home/appveyor/projects/jackett/src/Jackett.Common/Services/IndexerManagerService.cs:line 50
   at Jackett.Server.Services.ServerService.Initalize() in /home/appveyor/projects/jackett/src/Jackett.Server/Services/ServerService.cs:line 317
   at Jackett.Server.Helper.Initialize() in /home/appveyor/projects/jackett/src/Jackett.Server/Helper.cs:line 34
   at Jackett.Server.Startup.ConfigureServices(IServiceCollection services) in /home/appveyor/projects/jackett/src/Jackett.Server/Startup.cs:line 95
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Hosting.ConventionBasedStartup.ConfigureServices(IServiceCollection services)
   at Microsoft.AspNetCore.Hosting.Internal.WebHost.EnsureApplicationServices()
   at Microsoft.AspNetCore.Hosting.Internal.WebHost.Initialize()
   at Microsoft.AspNetCore.Hosting.WebHostBuilder.Build()
   at Jackett.Server.Program.Main(String[] args)

 System.BadImageFormatException: Could not load file or assembly 'YamlDotNet, Version=6.0.0.0, Culture=neutral, PublicKeyToken=ec19458f3c15af5e'. The module was expected to contain an assembly manifest.
File name: 'YamlDotNet, Version=6.0.0.0, Culture=neutral, PublicKeyToken=ec19458f3c15af5e'
   at Jackett.Common.Services.IndexerManagerService.InitCardigannIndexers(IEnumerable`1 path)
   at Jackett.Common.Services.IndexerManagerService.InitIndexers(IEnumerable`1 path) in /home/appveyor/projects/jackett/src/Jackett.Common/Services/IndexerManagerService.cs:line 50
   at Jackett.Server.Services.ServerService.Initalize() in /home/appveyor/projects/jackett/src/Jackett.Server/Services/ServerService.cs:line 317
   at Jackett.Server.Helper.Initialize() in /home/appveyor/projects/jackett/src/Jackett.Server/Helper.cs:line 34
   at Jackett.Server.Startup.ConfigureServices(IServiceCollection services) in /home/appveyor/projects/jackett/src/Jackett.Server/Startup.cs:line 95
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Hosting.ConventionBasedStartup.ConfigureServices(IServiceCollection services)
   at Microsoft.AspNetCore.Hosting.Internal.WebHost.EnsureApplicationServices()
   at Microsoft.AspNetCore.Hosting.Internal.WebHost.Initialize()
   at Microsoft.AspNetCore.Hosting.WebHostBuilder.Build()
   at Jackett.Server.Program.Main(String[] args)

```

-------------------------------

**Comment by garfield69** [comment]
possible duplicate of #5208

**Comment by ngosang** [comment]
@hhazeley could you test this? thank you!
https://github.com/Jackett/Jackett/issues/5208#issuecomment-546564391

**Comment by garfield69** [comment]
fix is available with stable jackett 0.12.898. but you will need to install manually and run the systemd install script anew.

1. open a terminal and shutdown the jackett service using `systemctl stop jackett.service`
2. download and extract the linux package that applies to your platform from https://github.com/Jackett/Jackett/releases/tag/v0.12.898
3. copy or move/replace all the files from the Jackett folder in the downloaded/extracted package to where ever you have previously put the jackett binaries. Typical locations are `/opt/Jackett` or `~/Jackett`
4. cd to your Jackett binary folder and install the service with `sudo ./install_service_systemd.sh` 
this will start the service (if it ran successfully)
5. access your jackett dashboard with your browser and untick the Disable auto Update check box and press the Apply Server Settings button.
